packages/gatsby-source-wordpress/src/steps/create-schema-customization/transform-fields/index.js

```diff
@@ -93,6 +88,7 @@ export const transformFields = ({
   fields,
   parentType,
   parentInterfacesImplementingTypes,
+  peek = false,
```

Comment:
had to add this "peek" option because this function is called during query building as well as schema customization, and running parts of it in query building was causing some caching to be wrong later in schema customization. peek just means it checks which fields in the provided field list will exist in Gatsby's schema without actually transforming those fields for schema customization